lxd/db/generate/db/method_v2.go

```diff
@@ -1004,7 +1004,9 @@ func (m *MethodV2) delete(buf *file.Buffer, deleteOne bool) error {
 
 	m.ifErrNotNil(buf, "fmt.Errorf(\"Fetch affected rows: %w\", err)")
 	if deleteOne {
-		buf.L("if n != 1 {")
+		buf.L("if n == 0 {")
+		buf.L(`        return api.StatusErrorf(http.StatusNotFound, "%s not found")`, lex.Camel(m.entity))
+		buf.L("} else if n > 1 {")
```

Comment:
Can we include the entity name in the error below too? Thanks

Reply:
Fixed.